src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -0,0 +1,119 @@
+
+from azure.cli.core.commands.parameters import tags_type, get_enum_type, resource_group_name_type, name_type, get_location_type, get_three_state_flag
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+
+# pylint: disable=line-too-long
+def load_arguments_eh(self, _):
+    from knack.arguments import CLIArgumentType
+    from azure.mgmt.eventhub.models.event_hub_management_client_enums import KeyType, AccessRights, EntityStatus, SkuName, SkuTier
+
+    rights_arg_type = CLIArgumentType(options_list=['--rights'], nargs='+', arg_type=get_enum_type(AccessRights), help='Authorization rule rights of type list')
+    key_arg_type = CLIArgumentType(options_list=['--key-name'], arg_type=get_enum_type(KeyType), help='specifies Primary or Secondary key needs to be reset')
+    event_hub_name_arg_type = CLIArgumentType(options_list=['--event-hub-name'], id_part='child_name_1', help='Name of the EventHub')
+
+    with self.argument_context('eventhubs') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('namespace_name', id_part='name', help='name of the Namespace')
+
+    with self.argument_context('eventhubs namespace exists') as c:
+        c.argument('namespace_name', arg_type=name_type, help='name of the Namespace')
+
+    with self.argument_context('eventhubs namespace') as c:
+        c.argument('namespace_name', arg_type=name_type, id_part='name', help='name of the Namespace')
+
+    with self.argument_context('eventhubs namespace create') as c:
+        c.argument('tags', arg_type=tags_type)
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(SkuName))
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('skutier', options_list=['--sku-tier'], arg_type=get_enum_type(SkuTier))
+        c.argument('capacity', type=int, help='Capacity for Sku')
+        c.argument('is_auto_inflate_enabled', arg_type=get_three_state_flag(), help='Value that indicates whether AutoInflate is enabled for eventhub namespace.')
+        c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+
+    with self.argument_context('eventhubs namespace update') as c:
+        c.argument('namespace_name', arg_type=name_type, id_part='name', help='name of the Namespace')
+        c.argument('tags', arg_type=tags_type)
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(SkuName))
+        c.argument('skutier', options_list=['--sku-tier'], arg_type=get_enum_type(SkuTier))
+        c.argument('capacity', type=int, help='Capacity for Sku')
+        c.argument('is_auto_inflate_enabled', arg_type=get_three_state_flag(), help='Value that indicates whether AutoInflate is enabled for eventhub namespace.')
```

Comment:
Here is an easy opportunity to shorten a name:
`--auto-inflate`.  Then change three state flag to use labels "enabled" and "disabled", like so:
`get_three_state_flag('enabled', 'disabled')`

Reply:
discussed with team, added prefix of enable to all Boolean parameters.